i2c-bus/index.d.ts

```diff
@@ -0,0 +1,400 @@
+// Type definitions for i2c-bus v1.0.0
+// Project: https://github.com/fivdi/i2c-bus
+// Definitions by: Jason Heard <https://github.com/101100>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module "i2c-bus" {
```

Comment:
remove this block.
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
We recommend `ProperModule` kind.